README.md

```diff
@@ -681,37 +681,40 @@ non-IO.
 
 # Developing Semian
 
-Semian requires a Linux environment.  We provide a [Vagrantfile](Vagrantfile)
-that installs MySQL, Redis and Ruby in an Ubuntu 16.04 virtual machine.  Use
+Semian requires a Linux environment.  We provide a [docker-compose](https://docs.docker.com/compose/) file
+that runs MySQL, Redis, Toxiproxy and Ruby in containers.  Use
 the steps below to work on Semian from a Mac OS environment.
 
+Prerequisites :
 ```bash
-# install Vagrant and VirtualBox
-$ brew cask install vagrant virtualbox
+# install docker-for-desktop
+$ brew cask install docker
+
+# install visual-studio-code (optional)
+$ brew cask install visual-studio-code
 
 # clone Semian
 $ git clone https://github.com/Shopify/semian.git
 $ cd semian
+```
 
-# build the virtual machine (can take a while for ruby to compile)
-$ vagrant up
+Developing :
 
-# shell in to the virtual machine
-$ vagrant ssh
+- Visual Studio Code
+  - Open semian in vscode
+  - Install recommended extensions (one off requirement)
+  - Click `reopen in container` (first boot might take about a minute)
+  
+  See https://code.visualstudio.com/docs/remote/containers for more details
 
-# start the toxiproxy-server inside the virtual machine
-$ toxiproxy-server > /dev/null &
-# TODO: provide a systemd unit for toxiproxy
-
-# run the tests inside the virtual machine
-$ cd /vagrant && bundle exec rake
-```
+- Everything else
+  - `$ docker-compose -f .devcontainer/docker-compose.yml run semian bash`
 
 Be careful not to run `bundle install` from Mac OS.  The folder is shared with
```

Comment:
Is this still true?
If so, why? Can't we just run `bundle install` inside the container, to avoid polluting the external environment?

Reply:
Took this out as its no longer affects the container.
Also changed the docker-compose file to start in `/workspace`, run `bundle install` and then `sleep infinity`.